### PR TITLE
Removed the word pool from the project

### PR DESCRIPTION
**Pull request by pix-art**
Changed the word pool everywhere, ran tests, and did some smoke tests.

This is gonna be fun :-)

### CODE REVIEW

File: src/Intracto/SecretSantaBundle/Resources/translations/messages.de.yml [hunk 1]
```diff
@@ -380,17 +380,17 @@ emails-pool_status:
 #        wishlist_filled: ?
 
 #    btn:
-#        poolstatus: ?
+#        partiestatus: ?
```
Comment (tvlooy) [inline src/Intracto/SecretSantaBundle/Resources/translations/messages.de.yml]: seems like a typo s/partie/party/

File: src/Intracto/SecretSantaBundle/Resources/views/Report/analytics.html.twig [hunk 2]
```diff
@@ -10,7 +10,7 @@
     </div>
 {% endif %}
 
-{% if google_data_pool is not empty %}
+{% if google_party_data is not empty %}
```
Comment (tvlooy) [inline src/Intracto/SecretSantaBundle/Resources/views/Report/analytics.html.twig]: this is a bad name, this data is not "party" related but google analytics data. We should completely avoid the word "pool" here and go for a more suitable name

**Comment by tvlooy** [comment]
@pix-art I made 2 notes. Can you fix these?

**Comment by pix-art** [comment]
I'll fix them first thing in the morning.

**Comment by tvlooy** [comment]
@pix-art can you merge conflicts and rebase? This can go in if it's ready to merge

**Pull request merged**